tutorials/physics/physics_introduction.rst

```diff
@@ -138,6 +140,32 @@ interact with. For example, the Player's settings would look like this:
 .. image:: img/player_collision_layers.png
 .. image:: img/player_collision_mask.png
 
+**Code Example**
+In function calls, layers are specified as a bitmask. Where a function enables
+all layers by default, the layer mask will be given as 0x7FFFFFFF. Your code
+can use binary, hexadecimal, or decimal notation for layer masks, depending 
+on your preference.
+
+The code equivalent of the above example where layers 1, 3 and 4 were enabled
+would be as follows : 
+
+:: _physics_introduction_collision_layer_code_example:
+
+    # Example : Setting mask value for enabling layers 1, 3 and 4
+
+    # Binary - set the bit corresponding to the layers you want to enable (1, 3, and 4) to 1, set all other bits to 0.
+    # Note : layer 20 is the first bit, layer 1 is the last. The mask for layers 4,3 and 1 is therefore
+    0b00000000000000001101 
+    # (This can be shortened to 0b1101)
+
+    # Hexadecimal equivalent (1101 binary converted to hexadecimal)
+    0x000D
+    # (This value can be shortened to 0xD)
+    
+    # Decimal - Raise each layer to be enabled (1, 3, and 4) to the power of 2 and add all the results together.
+    (1^2) + (3^2) + (4^2) = 13
```

Comment:
GDScript doesn't have an exponentation operator. I think it would be better to remove the decimal form entirely at this point.

Reply:
Corrected the documentation here - had the calculation backwards, and modified the document to reference the pow function for performing the exponential calculation.